mpmp/src/model/Player.java

```diff
@@ -6,368 +6,366 @@
 import java.util.Iterator;
 
 /**
- * Player represents spectators and the actual, active players. The info stored here exists in both
+ * Player represents spectators and the actual, actuve players. The info stored here exists in both
  * client and server and is kept up-to-date by the clientlist-update packet.
  *
  * @author Leander, oki
  */
 public class Player {
 
-	// XXX move to a new file of constants?
-	private static final int StartCash = 30000; // XXX value
-	private static final int Wage = 4000;  // XXX value
-	private static final int IncomeTax = 4000;  // XXX value
-	private static final int ExtraTax = 2000;  // XXX value
-	public static final int UnjailFee = 1000;
-	private static Player currentPlayer;
-
-	public enum Mode {
-
-		Spectator, Player
+    // XXX move to a new file of constants?
+    private static final int StartCash = 30000; // XXX value
```

Comment:
@dieechtenilente Explain to me how this line is tab-indented. You're such a derp :P